src/Symfony/Component/HttpClient/Tests/HttpClientTestCase.php

```diff
@@ -114,6 +114,16 @@ public function testReentrantBufferCallback()
         $response->getContent();
     }
 
+    public function testInfoOnCanceledResponse()
+    {
+        $client = $this->getHttpClient(__FUNCTION__);
+
+        $response = $client->request('GET', 'http://localhost:8057');
+        $response->cancel();
+
+        $this->assertTrue($response->getInfo('canceled'));
```

Comment:
we need another check with assertFalse (in the same test case that's alright)

Reply:
Fixed.